### PR TITLE
🎱 Fields: `Finished`, `Successful`

### LINKED ISSUE #183: MetaflowRunsView: Finished icon
Change `Finished=False` to spinning yellow, or other "in progress" indicator 🔄

### PR DESCRIPTION
**Pull request by mrharpo**
Adds custom bool fields for Finished, Successful

# Finished
- `True` Blue check ✔️  
- `False` Grey clock 🕐

# Successful
- `True` Green circle check ✅ 
- `False` Red circle X ❌

Screen Shot 2023-10-27 at 10 54 24

Closes #183

### CODE REVIEW

File: static/js/custom-render.js [hunk 1]
```diff
@@ -21,4 +21,30 @@ Object.assign(render, {
...
+        d === true
+          ? `<div class="p-1"><span class="text-center text-info"><i class="fa-solid fa-check fa-lg"></i></span></div>`
+          : `<div class="p-1"><span class="text-center text-secondary"><i class="fa-solid fa-clock fa-lg"></i></span></div>`
+      )
+      .join("")}</div>`;
+  },
+  successful: function render(data, type, full, meta, fieldOptions) {
+    if (data == null) return null_column();
+    if (Array.isArray(data) && data.length == 0) return empty_column();
+    data = Array.isArray(data) ? data : [data].map((d) => d === true);
```
Comment (afred) [inline static/js/custom-render.js:40]:
```
data = Array.isArray(data) ? data : [data].map((d) => d === true);
```
☝️ I think all this needs to do is ensure data is an Array, i.e...
```
data = Array.isArray(data) ? data : [data]
```
Checking for whether the values `=== true` is happening in the interpolated return value right below, so no need to do it twice.

File: static/js/custom-render.js [hunk 2]
```diff
@@ -21,4 +21,30 @@ Object.assign(render, {
       ? `<img src="${data.location}" style="max-height:150px;" loading="lazy">`
       : null
   },
+  finished: function render(data, type, full, meta, fieldOptions) {
+    if (data == null) return null_column();
+    if (Array.isArray(data) && data.length == 0) return empty_column();
+    data = Array.isArray(data) ? data : [data].map((d) => d === true);
+    if (type != "display") return data.join(",");
```
Comment (afred) [inline static/js/custom-render.js:28]: ☝️ what's the use case for when `type != "display"` and what is the visual effect of `return data.join(",")`?

Comment (mrharpo) [reply static/js/custom-render.js:28]: 🤷 Directly copied from https://github.com/WGBH-MLA/starlette-admin/blob/855a71ccf7b7a4fa6ccd3026a9bf593eccb18d7c/starlette_admin/statics/js/render.js#L42-L54

I think it's for

**Review by afred — APPROVED** [review]
👍

**Pull request merged**